### ISSUE TITLE
Add documentation link for AvoidPropertySelfAssignment CA2245

### ISSUE THREAD

**Issue by mclayton7** [dotnet/roslyn-analyzers#2610]
#### Analyzer package

[Microsoft.CodeQuality.Analyzers](https://www.nuget.org/packages/Microsoft.CodeQuality.Analyzers)

#### Package Version
Latest

#### Diagnostic ID

CA2245

#### Expected behavior
Should have link to documentation.

#### Actual behavior
Doesn't have link (link doesn't exist yet).

#### Blocked by 
https://github.com/MicrosoftDocs/visualstudio-docs/issues/3510

**Comment by mavasani** [comment]
Duplicate of https://github.com/dotnet/roslyn-analyzers/issues/2611

@mclayton7 Sorry, we both happened to file the same issue simultaneously.
